docs/getting_started/hopsworksai/guides/databricks_quick_start_azure.rst

```diff
@@ -0,0 +1,190 @@
+Databricks Quick Start (Azure)
+==============================
+
+This guide shows how to quickly get started with using the Feature Store from Databricks on Azure.
+
+.. contents:: :local:
+
+Step 1: Setting up VPC peering between Hopsworks and Databricks
+---------------------------------------------------------------
+VPC peering between the Hopsworks and the Databricks virtual network is required to be able to connect
```

Comment:
`VPC` is Amazon terminology. Azure's equivalent is `VNet` https://docs.microsoft.com/en-us/azure/virtual-network/virtual-networks-overview

Reply:
fixed